### ISSUE TITLE
Modlog paging is broken

### ISSUE THREAD

**Issue by Zefyro** [InsanityBot/insanitybot-v2#64]
**Describe the bug**
modlog paging is broken. (thats probably as much as I can describe it.)

**To Reproduce**
use the `modlog` command 

**Expected behaviour**
Expected after using previous page "button" for it to return to the previous modlog page which did not happen.

**Screenshots**
![Screenshot](https://user-images.githubusercontent.com/71092443/113576276-ed0ee780-9627-11eb-800f-5172b5be9d93.png)

**Host system (please complete the following information):**
 - OS: Debian 10, Pterodactyl, ptero version 1.2.2
 - Bot version: 2.0.0-dev.00019

**Additional context**

**Comment by akiraveliara** [comment]
Are there any errors in console?

**Comment by Zefyro** [comment]
> Are there any errors in console?

Oh yeah, forgot to add that thing.

**Comment by akiraveliara** [comment]
stacktrace:
~~~
[2021-04-05 14:56:08 +02:00] [0   /            ] [Error] System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.Collections.Generic.List`1.Enumerator.MoveNextRare()
   at System.Collections.Generic.List`1.Enumerator.MoveNext()
   at System.Linq.Enumerable.WhereListIterator`1.MoveNext()
   at InsanityBot.Commands.Moderation.Modlog.ModlogMessageTracker.AddTrackedMessage(MessageTrackerEntry entry)
   at InsanityBot.Commands.Moderation.Modlog.Modlog.PagedModlogHandler(MessageReactionAddEventArgs args)
~~~

**Comment by akiraveliara** [comment]
fixed as of b6adbc7/2.0.0-dev.00020
